App-971 : refactoring DAO factory (#113)
aragon/core: packages/contracts/CHANGELOG.md

### Added

> - Added test util functions such as `deployPluginRepoRegistry`, `deployPluginSetupProcessor`, `deployPluginRepoFactory` and `filterEvents`.

Reviewer: ```suggestion
- Added utility functions (`deployPluginRepoRegistry`, `deployPluginSetupProcessor`, `deployPluginRepoFactory`, and `filterEvents`) to the test suite.
```

Author: amended

### Changed

> - Refactored 'DAOFactory`to use`PluginSetupProcessor`.

Reviewer 2: wrong quote simbol. should be a \` and not a `'`

Author: thanks
